.github/workflows/dev.yml

```diff
@@ -45,9 +45,9 @@ jobs:
       - name: Populate .env with additional vars
         run: |
           cp ./.github/.github.env .env
-          echo AWS_ACCOUNT=${{ secrets.AWS_ACCOUNT }} >> .env
-          echo AWS_REGION=${{ secrets.AWS_REGION }} >> .env
-          echo AWS_DEFAULT_REGION=${{ secrets.AWS_REGION }} >> .env
+          echo \AWS_ACCOUNT=${{ secrets.AWS_ACCOUNT }} >> .env
```

Comment:
Do we need these backslashes?

Reply:
Yeah, there's a formatting issue with the .env file where AWS_ACCOUNT was ending up on the end of the last line of the env file; this was causing the regex tests to fail.

Reply:
Hmm not entirely sure how/why this has now suddenly cropped in as an issue? Think we probably want to get the bottom of this as not ideal having to add this backslash in, might be we have to wrap our env variables with quotes or similar?